tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Media/Media.spec.ts

```diff
@@ -0,0 +1,181 @@
+    // Arrange
+    await umbracoApi.media.ensureNameNotExists(folderName);
+
+    // Act
+    await umbracoUi.media.goToSection(ConstantHelper.sections.media);
+    await umbracoUi.media.clickCreateMediaItemButton();
+    await umbracoUi.media.clickMediaTypeWithNameButton('Folder');
+    await umbracoUi.media.enterMediaItemName(folderName);
+    await umbracoUi.media.clickSaveButton();
+
+    // Assert
+    await umbracoUi.media.isSuccessNotificationVisible();
+    await umbracoUi.media.isTreeItemVisible(folderName);
+    expect(await umbracoApi.media.doesNameExist(folderName)).toBeTruthy();
+
+    // Clean
+    await umbracoApi.media.ensureNameNotExists(folderName);
+  });
+
+  test('can delete a folder', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    await umbracoApi.media.ensureNameNotExists(folderName);
+    await umbracoApi.media.createDefaultMediaFolder(folderName);
+    await umbracoUi.media.goToSection(ConstantHelper.sections.media);
+    await umbracoApi.media.doesNameExist(folderName);
+
+    // Act
+    await umbracoUi.media.clickActionsMenuForName(folderName);
+    await umbracoUi.media.clickDeleteButton();
+    await umbracoUi.media.clickConfirmToDeleteButton();
+
+    // Assert
+    await umbracoUi.media.isTreeItemVisible(folderName, false);
+    expect(await umbracoApi.media.doesNameExist(folderName)).toBeFalsy();
+  });
+
+  test('can create a folder in a folder', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    const parentFolderName = 'ParentFolder';
+    await umbracoApi.media.ensureNameNotExists(parentFolderName);
+    await umbracoApi.media.createDefaultMediaFolder(parentFolderName);
+    await umbracoUi.media.goToSection(ConstantHelper.sections.media);
+
+    // Act
+    await umbracoUi.media.clickActionsMenuForName(parentFolderName);
+    await umbracoUi.media.clickCreateModalButton();
+    await umbracoUi.media.clickExactLinkWithName('Folder');
+    await umbracoUi.media.enterMediaItemName(folderName);
+    await umbracoUi.media.clickSaveButton();
+
+    // Assert
+    await umbracoUi.media.isSuccessNotificationVisible();
+    await umbracoUi.media.isTreeItemVisible(parentFolderName);
+    await umbracoUi.media.clickCaretButtonForName(parentFolderName);
+    await umbracoUi.media.isTreeItemVisible(folderName);
+
+    // Clean
+    await umbracoApi.media.ensureNameNotExists(parentFolderName);
+  });
+
+  test('can search for a media file', async ({umbracoApi, umbracoUi}) => {
+    // Arrange
+    const secondMediaFile = 'SecondMediaFile';
+    await umbracoApi.media.ensureNameNotExists(secondMediaFile);
+    await umbracoApi.media.createDefaultMedia(mediaFileName, mediaTypeName);
+    await umbracoApi.media.createDefaultMedia(secondMediaFile, mediaTypeName);
+    await umbracoUi.media.goToSection(ConstantHelper.sections.media);
+
+    // Act
+    await umbracoUi.media.searchForMediaItemByName(mediaFileName);
+
+    // Assert
+    await umbracoUi.media.doesMediaCardsContainAmount(1);
```

Comment:
It would be better if you get amount of media card from API instead of the hard code.